src/lambda/wizards/samDeployWizard.ts

```diff
@@ -134,6 +136,9 @@ export class DefaultSamDeployWizardContext implements SamDeployWizardContext {
             'View Documentation'
         )
     )
+    // this represents a cache that is created once per wizard load.
+    // if we wish to change the frequency of this cache, remove `readonly` so you can reassign later
+    private readonly regionBucketMap: Map<string, string[]> = new Map<string, string[]>()
```

Comment:
`regionBucketMap` does not make it apparent what the key or value is. "Map" may not be necessary as part of the name.